src/main/scala/example/Book.scala

```diff
@@ -0,0 +1,14 @@
+package example
+
+import io.circe._, io.circe.parser._
+import scala.util.{Try, Success, Failure}
+
+case class Book(isbn: String)
+object Book {
+  def fromVolume(vol: Volume): Either[TransformError, Book] = {
+    Try(vol.volumeInfo.industryIdentifiers.head.identifier) match {
```

Comment:
`Try` blocks give me pause, as it indicates that it's swallowing an exception, which we should trying to stay away from if at all possible. Is there a way the `val.volumeInfo.Industry.....`. unpacking could return an `Option` which we could then `match` on in the next line? So like, `case Some(val) => Book(val)` and `case None => TransformError(...)`

Reply:
I understand the apprehension of using the try block. The unpacking cannot _automatically_ return an option, because that's not what my types are. I could change the properties on the case classes in the Json structures files to use Options, but I'm not sure if that's what you meant. I could also use [optics](https://www.scala-exercises.org/circe/Optics) which i think would give me a better syntax for this, but that would mean my current case classes wouldn't be very helpful outside of encoding the json for testing.  I might also end up deleted some functions.

Reply:
Yeah I meant reworking the types to make use of Option. It’s the safe choice to use when something might or might not return a value.

Reply:
If i converted all the fields to be options then the grabbing of the isbn would look something like this 
```
vol
      .volumeInfo
      .industryIdentifiers
      .flatMap(_.lift(0))
      .flatMap(_.identifier)
```

Reply:
I'm not sure if everything needs to be an option, just fields that may or may not be returned by the google books response (I'm not sure what shape that response takes). 

But whether it's an `Option` or otherwise, the `Try` is a blocker for me for this pr.